src/components/Text.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import styled from "@wigxel/react-components";
+import { useLayout } from '../libs/LayoutStore'
+
+const Heading1 = styled.h1`
+  font-size: 2em;
+  
+  span {
+    font-size: 2.3em;
+    
+  }
+`
+
+const Heading3 = styled.h3`
+  font-size: 1.8em;
+`
+
+const P = styled.p`
+  font-size: 1.2rem;
+`
+
+export const H1 = props => {
```

Comment:
You don't have to be extract the props you want here. Try this instead
```js
const H1 = Heading1
```
Instead of being redundant.

Reply:
Do the same for the rest of this file and any other place where you may have dont it.